src/app/app.module.ts

```diff
@@ -1,18 +1,17 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { ReactiveFormsModule } from '@angular/forms';
+import { FormsModule } from '@angular/forms'; // <-- import this module
```

Comment:
I would removed the comment.

Reply:
done!. will be more careful regarding comments.